layouts/partials/blocks/team.html

```diff
@@ -0,0 +1,60 @@
+{{ with .block }}
+  <section class="bg-s-2 py-10 px-2">
+    <div class="mx-auto my-14 w-full max-w-[1248px]">
+      <h2
+        class="mb-10 text-4xl font-black uppercase leading-none text-s-6 md:text-6xl"
+      >
+        {{ .title }}
+      </h2>
+
+      {{ with .staff }}
+        <ul
+          class="mx-auto my-14 flex w-full max-w-[1248px] flex-wrap justify-between"
+        >
+          {{ range . }}
```

Comment:
You might be able to use the  helper {{ $author := partial "helper/get-author" $name }} so you can just pull this from the staff data directly. See https://github.com/spotlightpa/poor-richard/blob/master/layouts/partials/tw/main.html#L141 . Then again maybe we want a separate bio/headshot here.

Reply:
Updated: https://github.com/spotlightpa/poor-richard/pull/338#discussion_r1083246589

> You might be able to use the helper {{ $author := partial "helper/get-author" $name }} so you can just pull this from the staff data directly. See https://github.com/spotlightpa/poor-richard/blob/master/layouts/partials/tw/main.html#L141 . Then again maybe we want a separate bio/headshot here.